library/sbi-rt/src/fwft.rs

```diff
@@ -0,0 +1,57 @@
+//! Chapter 18. SBI Firmware Features Extension (EID #0x46574654 "FWFT")
+
+use crate::binary::{sbi_call_1, sbi_call_3};
+use sbi_spec::{
+    binary::SbiRet,
+    fwft::{EID_FWFT, SET, GET},
+};
+
+/// Set the value of a specific firmware feature
```

Comment:
```suggestion
/// Set the configuration value of a specific firmware feature.
```

Ends with `.`; according to the standard, the FWFT extesion sets a 'configuration value' for firmware features.